.travis.yml

```diff
@@ -9,7 +9,7 @@ cache:
 os: linux
 language: python
 python:
-  - "3.6"
+  - "3.7"
```

Comment:
We can create this as a matrix (that is support both 3.6 and 3.7) but I think supporting 3.7 should suffice.